reinovo_control/nav_goal nav_goal
---
bool success
string message